### ISSUE TITLE
Do duplicated projects count as new projects?

### ISSUE THREAD

**Issue by MJWainwright** [epistemonikos/isoq#197]
When we get a count of how many new projects have been created in iSoQ does this include duplicated projectsi.e. copies of existing projects?

**Comment by damian-garrido** [comment]
yes, its include the duplicated, because are considered with new id
